ibis/backends/tests/test_generic.py

```diff
@@ -1441,7 +1438,18 @@ def test_try_cast_func(con, from_val, to_type, func):
         ##################
         ### POSITIVE start
         # no stop
-        param(slice(3, 0), lambda _: 0, id="[3:0]"),
+        param(
+            slice(3, 0),
+            lambda _: 0,
+            id="[3:0]",
+            marks=[
+                pytest.mark.never(
+                    ["impala"],
+                    raises=ImpalaHiveServer2Error,
+                    reason="impala doesn't support OFFSET without ORDER BY",
```

Comment:
Could placing an order by null work here?

Reply:
Hm, perhaps, but I didn't try it.

Reply:
I leave this for later.